dev/conformance/conformance-tests/set-st-merge-nowrite.json

```diff
@@ -19,15 +19,16 @@
           "database": "projects/projectID/databases/(default)",
           "writes": [
             {
-              "transform": {
-                "document": "projects/projectID/databases/(default)/documents/C/d",
-                "fieldTransforms": [
-                  {
-                    "fieldPath": "b",
-                    "setToServerValue": "REQUEST_TIME"
-                  }
-                ]
-              }
+              "update": {
+                "fields": {},
+                "name": "projects/projectID/databases/(default)/documents/C/d"
+              },
```

Comment:
This is missing an updateMask - but I would prefer if we could keep the original Protobuf format here.